ansible/vars/icds/icds_public.yml

```diff
@@ -0,0 +1,259 @@
+SITE_HOST: 'www.icds-cas.gov.in'
+#SITE_HOST: 'cas.commcarehq.org'
+CAS_SITE_HOST: 'cas.commcarehq.org'
+#NO_WWW_SITE_HOST: 'cas.commcarehq.org'
+NO_WWW_SITE_HOST: 'icds-cas.gov.in'
+deploy_env: icds
+tableau_server: 10.247.24.11:80
+
+# these 2 must be replaced with the real icds cert when we get it from NIC
+nginx_combined_cert_file: '../config/{{ deploy_env}}/ssl/cas.commcarehq.org.combined.crt'
+apache2_key_file: '../config/{{ deploy_env }}/ssl/cas.commcarehq.org.key'
+
+cas_nginx_combined_cert_file: '../config/{{ deploy_env}}/ssl/cas.commcarehq.org.combined.crt'
+cas_key_file: '../config/{{ deploy_env }}/ssl/cas.commcarehq.org.key'
+
+active_sites:
+  cchq_ssl: True
+  cchq_redirect: True
+  cchq_http: True
+  cchq_http_redirect: True
+  cchq_http_j2me: False
+  commtrack_ssl: False
+  commtrack_http: False
+  hq_status: False
+  tableau: False
+  icds_tableau: False
+  dhis: False
+  wiki: False
+  wiki_http: False
+  motech: False
+  motech1: False
+  motech2: False
+  riakcs: True
+  cas_ssl: True
+
+riak_ring_size: 128
+
+proxy_type: 'nginx'
+primary_ssl_env: "icds"
```

Comment:
I think this will cause `web0` to have two `default_server` which I think will cause an nginx error.

https://github.com/dimagi/commcarehq-ansible/blob/d76abb7e2ccdcc28ed1a873be3bd734df4fc4b0c/ansible/roles/nginx/vars/cas_ssl.yml#L15

https://github.com/dimagi/commcarehq-ansible/blob/d76abb7e2ccdcc28ed1a873be3bd734df4fc4b0c/ansible/roles/nginx/vars/cchq_ssl.yml#L13

https://github.com/dimagi/commcare-hq-deploy/blob/master/fab/inventory/icds#L37-L41

I think one of those will need to be changed as well, but not sure which one it should be. Have you tested phones submitting to both cas and icds urls? I'd be surprised if both worked due to the phones not supporting SNI. (I think the goal is to move new apks to use the icds url? If not you can probably ignore testing that, and change this PR to make default_server work differently in `cchq_ssl.yml`